metaworld/envs/env_util.py

```diff
@@ -1,6 +1,6 @@
 import os
 
-ENV_ASSET_DIR = os.path.join(os.path.dirname(__file__), 'assets')
+ENV_ASSET_DIR = os.path.join(os.path.dirname(__file__), 'assets_updated')
```

Comment:
I created a helper function that allows this file to retain compatibility with v1 envs. For new XML, you'll just have to pass an extra `True` boolean argument to `get_asset_full_path()`

Reply:
Ah yes I'll fix this to use your helper.